clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -3148,6 +3158,7 @@ static void RenderFloatingPointOptions(const ToolChain &TC, const Driver &D,
                                  ComplexArithmeticStr(RangeVal));
         }
       }
+      LastComplexRangeOption = "-fcomplex-arithmetic";
```

Comment:
`Arg::getSpelling` returns the spelling. Don't write the string literal separately.